product_packaging_container_deposit/models/stock_package_type.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PackageType(models.Model):
+    _inherit = "stock.package.type"
+
+    container_deposit = fields.Many2one("product.product")
```

Comment:
```suggestion
    container_deposit_product_id = fields.Many2one("product.product")
```
and should have a domain according to the description

Reply:
this is not fixed

Reply:
For the domain to apply probably you mean that according to description:

> The deposit product should be a service product.